src/main/resources/lib/game-notifications.js

```diff
@@ -1,67 +1,70 @@
 
-var OFFICE_LEAGUE_GAME_EVENT_ID = storeLib.OFFICE_LEAGUE_GAME_EVENT_ID;
+const OFFICE_LEAGUE_GAME_EVENT_ID = storeLib.OFFICE_LEAGUE_GAME_EVENT_ID;
 
 exports.setupPushNotifications = function () {
+    const baseUrl = portal.pageUrl({ path: "/", type: "absolute" });
+
     eventLib.listener({
-        type: 'custom.' + OFFICE_LEAGUE_GAME_EVENT_ID,
+        type: "custom." + OFFICE_LEAGUE_GAME_EVENT_ID,
         localOnly: true,
         callback: function (event) {
             var gameId = event.data.gameId;
             var leagueId = event.data.leagueId;
 
             sendGameNotifications(gameId, leagueId);
-        }
+        },
     });
-};
-
-var sendGameNotifications = function (gameId, leagueId) {
-    storeLib.refresh();
-    var game = storeLib.getGameById(gameId);
-    if (!game) {
-        return;
-    }
-    var league = storeLib.getLeagueById(leagueId);
-    if (!league) {
-        return;
-    }
 
-    var playerIds = storeLib.getLeaguePlayersByLeagueId(leagueId, 0, -1).hits.map(function (lp) {
-        return lp.playerId;
-    });
-    log.info('League player Ids: ' + playerIds.join(','));
+    var sendGameNotifications = function (gameId, leagueId) {
+        storeLib.refresh();
+        const game = storeLib.getGameById(gameId);
+        if (!game) {
+            return;
+        }
+        const league = storeLib.getLeagueById(leagueId);
+        if (!league) {
+            return;
+        }
 
-    var baseUrl = app.config['officeleague.baseUrl'] || 'http://localhost:8080/portal/draft/office-league/app';
-    var url = baseUrl + '/games/' + game._id;
+        const playerIds = storeLib
+            .getLeaguePlayersByLeagueId(leagueId, 0, -1)
+            .hits.map(function (lp) {
+                return lp.playerId;
+            });
+        // log.info('League player Ids: ' + playerIds.join(','));
 
-    var points = [].concat(game.points || []);
-    // if (game.finished) {
-    //     log.info('Sending push notifications for ended game: ' + gameId);
-    //
-    //     storeLib.sendPushNotification({
-    //         playerIds: playerIds,
-    //         text: "A game has ended in '" + league.name + "'",
-    //         url: url
-    //     });
-    //
-    // } else
-    if (points.length === 0) {
-        log.info('Sending push notifications for new game: ' + gameId);
+        const url = baseUrl + "/games/" + game._id;
 
-        storeLib.sendPushNotification({
-            playerIds: playerIds,
-            text: "A game has started in '" + league.name + "'",
-            url: url
-        });
+        var points = [].concat(game.points || []);
+        // if (game.finished) {
+        //     log.info('Sending push notifications for ended game: ' + gameId);
+        //
+        //     storeLib.sendPushNotification({
+        //         playerIds: playerIds,
+        //         text: "A game has ended in '" + league.name + "'",
+        //         url: url
+        //     });
+        //
+        // } else
```

Comment:
What's the consensus on leaving commented out code in the source file at Enonic? I'm used to having a fairly strict 'remove it' policy (it can always be retrieved via git later), but I've seen it around a fair bit.

Reply:
Yeah. The game notifications do not work atm. Tried to fix that, no luck so far. I don't know how we want the notifications to work. So that code might be used again. We don't have any policy on it. I usually remove it if i know we will probably not ever use it. In this case its a fair chance we will use it so i left it.

Reply:
Ait, gotcha! 🙆🏾  Did the game notifications work in the previous version (before making it a PWA)? Do you know what broke them?

Reply:
No to my knowledge they never worked. Only on localhost eg. developer environment.